crates/engine/config/src/lib.rs

```diff
@@ -190,7 +190,7 @@ where
         type Value = FederatedGraph;
 
         fn visit_str<E: serde::de::Error>(self, v: &str) -> Result<Self::Value, E> {
-            federated_graph::from_sdl(v).map_err(E::custom)
+            FederatedGraph::from_sdl(v).map_err(E::custom)
```

Comment:
I just want to not have `FederatedGraph::from_sdl()` and `from_sdl()` as a free function in the public API. Not sure picking this one is the right call from an ergonomics perspective.

Reply:
I prefer `FederatedGraph::from_sdl(v)` because I'll have that type imported anyway already.